theano/tensor/nnet/tests/test_nnet.py

```diff
@@ -1617,3 +1618,19 @@ def test_h_softmax():
     
     assert(fun_output_tg(x_mat, y_mat).shape == (batch_size,))
     assert(fun_output(x_mat).shape == (batch_size, output_size))
+
+
+def test_elu():
+    x = matrix('x')
+    seed = theano.tests.unittest_tools.fetch_seed()
+    rng = numpy.random.RandomState(seed)
+    X = rng.randn(20, 30).astype(config.floatX)
+
+    # test the base case, without custom alpha value
+    y = elu(x).eval({x: X})
+    assert numpy.allclose(y, numpy.maximum(X, 0) + (numpy.exp(numpy.minimum(X, 0))-1))
+
+    # test for different constant alpha values
+    for alpha in 0, 1.5, 2, -1, -1.5, -2:
+        y = elu(x, alpha).eval({x: X})
+        assert numpy.allclose(y, numpy.maximum(X, 0) + alpha * (numpy.exp(numpy.minimum(X, 0))-1))
```

Comment:
use

utt.assert_allclose(y, ...)

this will generate an assert with a better error message in case of problems.